tests/test_config.py

```diff
@@ -54,17 +57,51 @@ def test_get_config_invalid():
     blabla:
         - asdf
         - adfs
-
     nomodel:
-      name: bla
-
+        name: bla
+    
     api:
-      name: bla
+        name: bla
     """
+    mo = mock.mock_open(read_data=test_file_invalid)
+    mo.return_value.name = "./foobar.yml"
     with mock.patch(
         "%s.open" % config.__name__,
-        mock.mock_open(read_data=test_file_invalid),
-        create=True,
+            mo,  # <-- use our mock variable here
+            create=True,
     ):
         with pytest.raises(ValueError, match="does not contain"):
             _ = config.get_validated_config("lalala")
+
+
+test = b"""
+       
+     xob10:
+        type: oracle
+        
+     """
+test_file_yaml = b"""
+
+    dbms: !include test
+
+    model:
+        name: my_model
+    
+    api:
+        name: my_api
+    """
+
+
+@mock.patch('builtins.open', new_callable=mock.mock_open, read_data=test_file_yaml)
+def test_yaml_include(mo):
+    """Test include sub config files."""
+    handlers = (mo.return_value, mock.mock_open(read_data=test).return_value,)
```

Comment:
Looks like the error can be solved the same way as with the other tests -- by assigning the file name to the mocked handler. Try this:
```suggestion
    mo2 = mock.mock_open(read_data=test)
    mo2.return_value.name = "test"
    handlers = (mo.return_value, mo2.return_value)
```
And then maybe move the line `mo.return_value.name = "./foobar.yml"` to the top in order to make the order more logical.